pom.xml

```diff
@@ -53,6 +53,8 @@
     <module>hudi-integ-test</module>
     <module>packaging/hudi-integ-test-bundle</module>
     <module>hudi-examples</module>
+    <module>hudi-flink-writer</module>
```

Comment:
could we just call this `hudi-flink` instead of hudi-flink-writer. It gives us the change to also build the reading support into the same module/bundle?

Reply:
> could we just call this `hudi-flink` instead of hudi-flink-writer. It gives us the change to also build the reading support into the same module/bundle?

yes, good idea